store/context/transactionContext.js

```diff
@@ -0,0 +1,93 @@
+///Data
+import React, { createContext, useContext } from 'react';
+import useReducerWithLogger from './logger';
+import axios from 'axios';
+
+export const TransactionsDataContext = createContext();
+export const useTransactionsDataContext = () => useContext(TransactionsDataContext);
+
+///gerekli ilk stateler buraya yazılacak
```

Comment:
Remove these comments or write in English if they are necessary

Reply:
I removed all comment lines